pages/docs/reference/generics.md

```diff
@@ -385,6 +385,8 @@ fun <T> copyWhenGreater(list: List<T>, threshold: T): List<String>
 
 </div>
 
+A where clause creates an intersection type; each sub-clause must be satisfied.  In the above example, the threshold and all list items must implement *both* CharSequence and Comparable.
```

Comment:
Hi!
Thanks for your addition.
I suggest minor changes for better formatting and readability:

```suggestion
The passed type must satisfy all conditions of the `where` clause simultaneously. In the above example, the `T` type must implement *both* `CharSequence` and `Comparable`.
```

Regards, 
Pavel,
Kotlin technical writer.